components/MintPage/MintNow/MintMoreModal.tsx

```diff
@@ -0,0 +1,166 @@
+              Congrats!
+            </pre>
+            <pre
+              className="font-quicksand 
+                text-[15px] xl:text-[25px]
+                text-center 
+                xl:pb-[25px]
+                pb-[5px]
+                dark:text-black text-white"
+            >
+              Cre8or DNA Secured.
+            </pre>
+            <Media
+              link="/assets/Mint/MintNow/MintCoreModal/mint_avatar.svg"
+              blurLink="/assets/Mint/MintNow/MintCoreModal/mint_avatar.png"
+              type="image"
+              containerClasses="xl:w-[192.3px] xl:h-[192.3px]
+          w-[125px] h-[125px]"
+            />
+            <pre
+              className="font-quicksand 
+                text-[15px] xl:text-[25px]
+                text-center 
+                xl:py-[25px]
+                py-[15px]
+                leading-[90.3%]
+                dark:text-black text-white"
+            >
+              {`You have ${possibleMintCount || "no"} more\nmints available`}
+            </pre>
+            <div className={`${possibleMintCount ? "block" : "hidden"}`}>
+              <Button
+                id="mint_now"
+                className="!px-0 !py-0
+                  xl:!w-[192.3px] xl:!h-[50px] 
+                  !w-[150px] !h-[40px]
+                  !font-eigerdals font-bold !bg-black 
+                  text-[15px] xl:text-[20px] 
+                  !rounded-[10px]
+                  !text-black dark:!text-white
+                  dark:!bg-black !bg-white"
+                onClick={mintCre8or}
+              >
+                Mint More
+              </Button>
+              <pre
+                className="font-quicksand 
+                    text-[15px] xl:text-[20px]
+                    py-[10px]
+                    text-center"
+              >
+                OR
+              </pre>
+            </div>
+            {maxOfCre8or === lockedCntOfCre8or ? (
+              <Button
+                id="mint_now"
+                className="!px-0 !py-0
+            xl:!w-[260.3px] xl:!h-[50px] 
+            !w-[220px] !h-[40px]
+            !font-eigerdals font-bold !bg-black 
+            text-[15px] xl:text-[20px]  
+            !rounded-[10px]
+            !text-black dark:!text-white
+            dark:!bg-black !bg-white"
+                onClick={() => window.open("/profile", "_blank")}
+              >
+                Set up profile
+              </Button>
+            ) : (
+              <Button
+                id="mint_now"
+                className="!px-0 !py-0
+            xl:!w-[260.3px] xl:!h-[50px] 
+            !w-[220px] !h-[40px]
+            !font-eigerdals font-bold !bg-black 
+            text-[15px] xl:text-[20px]  
+            !rounded-[10px]
+            !text-black dark:!text-white
+            dark:!bg-black !bg-white"
+                onClick={() => window.open("/stake", "_blank")}
+              >
+                Enter warehouse
+              </Button>
```

Comment:
this will be used in a few of your modals, right?

might be best to abstract this to a shared `<MintModalCTAButtons />` component.

what do you think?